### ISSUE TITLE
Support for scala 2.12 projects

### ISSUE THREAD

**Issue by azakordonets** [scoverage/sbt-coveralls#92]
Hi, i'm trying to add scala 2.12 support for my [Library](https://github.com/azakordonets/fabricator) , but i'm having troubles with compiling it. It looks like there's no sbt-coveralls library for this scala version - do you plan do add it in the nearest future ? 
```
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  ::          UNRESOLVED DEPENDENCIES         ::
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  :: org.scoverage#scalac-scoverage-runtime_2.12;1.1.1: not found
[warn]  :: org.scoverage#scalac-scoverage-plugin_2.12;1.1.1: not found
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn]  Note: Unresolved dependencies path:
[warn]          org.scoverage:scalac-scoverage-runtime_2.12:1.1.1 ((scoverage.ScoverageSbtPlugin) ScoverageSbtPlugin.scala#L30)
[warn]            +- com.github.azakordonets:fabricator_2.12:2.1.4
[warn]          org.scoverage:scalac-scoverage-plugin_2.12:1.1.1 ((scoverage.ScoverageSbtPlugin) ScoverageSbtPlugin.scala#L30)
[warn]            +- com.github.azakordonets:fabricator_2.12:2.1.4
[trace] Stack trace suppressed: run last *:update for the full output.
[error] (*:update) sbt.ResolveException: unresolved dependency: org.scoverage#scalac-scoverage-runtime_2.12;1.1.1: not found
[error] unresolved dependency: org.scoverage#scalac-scoverage-plugin_2.12;1.1.1: not found
[error] Total time: 4 s, completed 21-Jul-2017 08:53:08
```

**Comment by gslowikowski** [comment]
Upgrade `sbt-scoverage` plugin version to `1.5.0`.

**Comment by azakordonets** [comment]
Thanks, for some weird reason it didn't work for me at first - i had this dependency in my plugin.sbt , but i was getting same error. I removed it and then add it again - after that error disappeared

**Comment by darrenleeweber** [comment]
Fails for me, so I landed here and ^^ doesn't work using sbt `1.0.4`
```
# project/plugins.sbt
addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")  # 1.5.0 can't be found, 1.5.1 works
addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.1.0")  # adding this fails
```
```
[warn] 	module not found: org.scoverage#sbt-coveralls;1.1.0
[warn] ==== typesafe-ivy-releases: tried
[warn]   https://repo.typesafe.com/typesafe/ivy-releases/org.scoverage/sbt-coveralls/scala_2.12/sbt_1.0/1.1.0/ivys/ivy.xml
[warn] ==== sbt-plugin-releases: tried
[warn]   https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/org.scoverage/sbt-coveralls/scala_2.12/sbt_1.0/1.1.0/ivys/ivy.xml
[warn] ==== local: tried
[warn]   /users/dlweber/.ivy2/local/org.scoverage/sbt-coveralls/scala_2.12/sbt_1.0/1.1.0/ivys/ivy.xml
[warn] ==== public: tried
[warn]   https://repo1.maven.org/maven2/org/scoverage/sbt-coveralls_2.12_1.0/1.1.0/sbt-coveralls-1.1.0.pom
[warn] ==== local-preloaded-ivy: tried
[warn]   /users/dlweber/.sbt/preloaded/org.scoverage/sbt-coveralls/1.1.0/ivys/ivy.xml
[warn] ==== local-preloaded: tried
[warn]   file:////users/dlweber/.sbt/preloaded/org/scoverage/sbt-coveralls_2.12_1.0/1.1.0/sbt-coveralls-1.1.0.pom
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.scoverage#sbt-coveralls;1.1.0: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn] 	Note: Some unresolved dependencies have extra attributes.  Check that these dependencies exist with the requested attributes.
[warn] 		org.scoverage:sbt-coveralls:1.1.0 (scalaVersion=2.12, sbtVersion=1.0)
[warn] 
[warn] 	Note: Unresolved dependencies path:
[warn] 		org.scoverage:sbt-coveralls:1.1.0 (scalaVersion=2.12, sbtVersion=1.0) (/data/src/streaming_metrics/project/plugins.sbt#L3-4)
[warn] 		  +- default:streaming_metrics-build:0.1-SNAPSHOT (scalaVersion=2.12, sbtVersion=1.0)
```
